shared/src/business/entities/caseWorksheet/CaseWorksheet.ts

```diff
@@ -6,6 +6,7 @@ export class CaseWorksheet extends JoiValidationEntity {
   public finalBriefDueDate?: string;
   public primaryIssue?: string;
   public statusOfMatter?: string;
+  public judgeUserId: string;
```

Comment:
Similar comment as for docket number in Correspondence.ts, above. We need the judgeUserId to upsert a CaseWorksheet, and this to me is a sign that it should be on the entity.

Before, curiously, we were validating that "CaseWorksheet" was ok ... and then _afterwards_ adding on the judgeUserId to store in the database.

Reply:
Validating and then modifying reminds me of this:

![ShootFirstGaryLarson](https://github.com/user-attachments/assets/8ba0e4c4-e9be-4d34-867d-83a915fbb442)